roles/map-server-docker-image/templates/Dockerfile.j2

```diff
@@ -10,18 +10,25 @@ WORKDIR {{map_server_root}}
 
 RUN echo "{{ansible_date_time.epoch}}" > build_date
 
-RUN npm install tessera
-RUN npm install tilelive-tmstyle
-RUN npm install tilelive-http
-
-RUN echo "{{ansible_date_time.epoch}}" > build_date
+RUN npm install tessera \
+  tilejson \
+  tilelive-http \
+  tilelive-otp-stops \
+  tilelive-tmsource \
+  tilelive-tmstyle \
+  tilelive-vector \
+  tilelive-xray
 
 RUN git clone {{map_server_style_git}} {{map_server_style}}
+RUN git clone {{vector_map_server_style_git}} {{vector_map_server_style}}
+
+RUN sed -i -e 's#source: "http://tiletest.kapsi.fi:8080/index.json"#source: "tilejson+http://localhost:{{map_server_exposed_port}}/hsl-vector-map/index.json"#' {{map_server_style}}/project.yml
```

Comment:
Could we move this index.json file to deploy project and send it to server with ansible?